### DISCUSSION TITLE
How to modify gas_limit

### DISCUSSION THREAD

**Discussion by bennyzhe** [cosmos/community#60] [Chain Builders (Cosmos SDK)] [answered]
Hi team:
    The default value of gas is 200000.
`      --gas string               gas limit to set per-transaction; set to "auto" to calculate sufficient gas automatically (default 200000)`

Can the gas parameter be modified through the gov proposal?

**Comment by julienrbrt** [comment]
The default gas value is defined in the client. When using the CLI, it is defined by the Cosmos SDK (https://github.com/cosmos/cosmos-sdk/blob/fc52ae5fdbf10f6780f02fec591f01dcb5c294b6/client/flags/flags.go#L18).

What are you trying to achieve?

**Reply by bennyzhe** [reply]
Hi @julienrbrt , Thanks a lot. 

I started a test node locally and sent a text proposal.

Because the text is too long, out of gas:
`raw_log: 'out of gas in location: WritePerByte; gasWanted: 200000, gasUsed: 204858: out of gas'`

So I was wondering, if a chain that is already online wants to modify the maximum gas value, whether it can be done by submit a proposal.
At present, it is not possible.

**Reply by julienrbrt** [reply, marked as answer]
Usually I always use `--gas auto` or `--gas-adjustment 1.5` when getting out of gas issues.

Bumping the default gas limit for CLI isn't the right solution as it won't work for each message. Best is to simulate the transaction.
